getPopsure/dirty-swan
# refactor(storybook): Upgrade to v8

### What this PR does
This PR upgrades storybook to v8 which also means getting rid of `react-scripts` and upgrading typescript to v5.

### Why is this needed?
`react-scripts` and node version was preventing us from upgrading some internal packages like `react-markdown`.

### How to test?
Run `yarn` and `yarn dev` to see it working. Testing other scripts like `yarn test` or `yarn build` could also be useful.


### Checklist:

  Or give a reason why this does not apply:
  Or give a reason why this does not apply:

### Browser support

My code works in the following browsers:
